test.sh

```diff
@@ -252,16 +252,16 @@ fi
 
 # Write the AWS credential file (deleted on script exit)
 if [ -f "${aws_creds_path}" ]; then
-    cat $aws_creds_path > $credsfile
+    aws_credential_file_path="${aws_creds_path}"
```

Comment:
Perhaps this is out of scope for this PR, but now we have:
- `aws_creds_path`
- `aws_credential_file_path`
- `credsfile`

and it is not obvious from their names how their purpose differs.

Reply:
I have renamed the variables in d95ac60 and 4a3dda1 to make the intent a bit clearer.